Mlem/Views/Tabs/Search/Search View.swift

```diff
@@ -43,6 +43,7 @@ struct SearchView: View {
         }
         .handleLemmyLinkResolution(navigationPath: $navigationPath)
         .searchable(text: getSearchTextBinding(), prompt: "Search for communities")
+        .disableAutocorrection(true)
```

Comment:
`disableAutocorrection` is deprecated - use `autocorrectionDisabled` instead 👍

Reply:
We might also want to specify `textInputAutoCapitalization(false)`, too.

Reply:
Good point 👍🏻
It wasn't marked as deprecated for me, shouldn't it do that? Either way, I fixed it. Thanks :)